src/main/java/nl/novi/techiteasycontroller/Model/Television.java

```diff
@@ -0,0 +1,4 @@
+package nl.novi.techiteasycontroller.Model;
+
+public class Television {
```

Comment:
Deze klasse is ook nog niet af. Voeg er nog attributen aan toe met getters/setters en eventueel nog een all args constructor